### PR TITLE
Import account creation doc by Corey

### PR DESCRIPTION
**Pull request by oskarth**
https://notes.status.im/6IN8H23dT9-xEM6Xov4Slw?view

@corpetty I'm cleaning up the specs repo, and I took the liberty to import your doc here. If you prefer to do it yourself, feel free to open a new PR and close this one.

### CODE REVIEW

File: x9.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Status Account Creation and Maintenance Specification
```
Comment (oskarth) [inline x9.md:1]: is there a simpler way to refer to 'creation and maintenance'? just use "Account" or 'lifecycle' or 'handling'?

Comment (oskarth) [reply x9.md:1]: It's a lot of words already,

Status
Acconut
Creation and
Maintenance
Specification

compared to

x3dh
spec

or

double ratchet
spec

File: x9.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Summary
+
+The core concept of an account in Status is a set of cryptographic keypairs.  Namely, the combination of the following:
```
Comment (oskarth) [inline x9.md:5]: Should this be part of terminology?

File: x9.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        - following [EIP1581](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1581.md)
+    - DB encryption Key ($DBK$): `m/43'/60'/1581'/1'/0` (post Multiaccount integration)
+        - following [EIP1581](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1581.md)
+    - Status Wallet paths: `m/44'/60'/0'/0'/i` starting at `i=0`
+        - following [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+        - NOTE: this (`i=0`) is also the current (and only) path for Whisper key before Multiaccount integration
+
+### 1.2 X3DH Prekey bundle creation
+- A client must create an amount of X3DH prekey bundles, each defined by the following items:
+    - Identity Key: $IK$
```
Comment (oskarth) [inline x9.md:29]: same comment re TeX

File: x9.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- Ethereum address (future verification, currently the same base keypair)
+- 3 word mnemonic name
+- identicon
+- message signatures
+
+## 1 Initial Key Generation
+### 1.1 Public/Private Keypairs 
+- An ECDSA (secp256k1 curve) public/private keypair MUST be generated via a [BIP43](https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki) derived path from a [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) mnemonic seed phrase.
+- The default paths are defined as such:
+    - Whisper Chat Key ($IK$): `m/43'/60'/1581'/0'/0`  (post Multiaccount integration)
```
Comment (oskarth) [inline x9.md:19]: This is hard to reason about tbh (time dependent specs), lets make sure we simplify this. The goal of this should be v1 specs, and there multiaccount will be used.

Also, should we have a separate doc for multiaccount?

**Review by oskarth — COMMENTED** [review]
general comments

**Comment by oskarth** [comment]
I think it'd be good if we could merge a rough draft of this and then iterate with specific GH issues. This way we have something done and avoid getting in situation like before, where we had a few PRs open for months due to big initial scope. 

Lmk what you think @corpetty 

Also cc @decanus

File: x9.md (hunk 1 above)
Comment (corpetty) [reply x9.md:1]: Yes I hadn't gotten to the point of unifying all the terms. To keep it relatively general, `Account Specification` seems good.

File: x9.md (hunk 3 above)
Comment (corpetty) [reply x9.md:29]: I would like to come together on a set of variables for each of these things as they're referred to across the specs.  Might have to make a master list of them all to see what all we're using.

File: x9.md (hunk 4 above)
Comment (corpetty) [reply x9.md:19]: I don't think we should separate for multi-account, unless you want to separate the wallet spec from the chat spec.  In my eyes, the account encompasses all a user is and needs to do to participate in the network, at least in terms of keys.

**Comment by corpetty** [comment]
Thanks fine, I'll leave an approve review and we can start making specific issues around changing it.

**Review by corpetty — APPROVED** [review]
approval for getting a rough draft in, then iterating over issues.

**Pull request merged**